src/server/game/AI/SmartScripts/SmartAI.cpp

```diff
@@ -800,7 +800,8 @@ void SmartAI::AttackStart(Unit* who)
                 me->GetMotionMaster()->MovementExpired();
                 me->StopMoving();
             }
-            me->GetMotionMaster()->MoveChase(who, GetScript()->GetCasterActualDist() ? GetScript()->GetCasterActualDist() : GetScript()->GetActualCombatDist());
+            float range = GetScript()->GetCasterActualDist() > 0.f ? GetScript()->GetCasterActualDist() : GetScript()->GetActualCombatDist();
+            me->GetMotionMaster()->MoveChase(who, range > 0.f ? ChaseRange(range) : std::optional<ChaseRange>());
```

Comment:
```std::optional<ChaseRange>()```
To
```Optional<ChaseRange>()```

Reply:
It's `std::optional` in `MoveChase` call - I didn't want to change that.